### ISSUE TITLE
Calendar date picker traps focus on Shift+Tab

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#9568]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=29728&p=150444#p150444)

"Hello!

I came across this issue during accessibility testing. It is easy to reproduce in this demo:

https://bryntum.com/products/calendar/examples/basic/

Put focus into the "Filter Events" input field below the date picker in the sidebar.
Press Shift+Tab to move into the date picker (there is no focus indication at this point, which is another accessibility issue).
Press Shift+Tab to move into the date picker's header. The "Next Year" button is focused.
Press Shift+Tab two more times. The "Year" button is focused without indication (yet another accessibility issue).
Keep pressing Shift+Tab, the focus moves to the "Previous Month" button, then to the "Previous Year" button, then disappears (at this point it is on the selected date), then it moves to the "Next Year" button again, effectively trapping the focus in the date picker.

Pressing Tab without Shift allows focus to escape the trap and move into the "Filter Events" input field, so the trap is only applied when moving backwards."

**Comment by ghulamghousdev** [comment]
Another report here: https://forum.bryntum.com/viewtopic.php?p=153503#p153503
